src/activities/multiplication-chart/MultiplicationChart.qml

```diff
@@ -0,0 +1,312 @@
+                   state: "default"
+                   states:[
+                       State {
+                           name: "default"
+                           PropertyChanges { target: dotsGridCol; color: "white"}
+                           PropertyChanges { target: dotsGridCol; clickedFlagCol: false}
+                       },
+                       State {
+                           name: "active"
+                           PropertyChanges { target: dotsGridCol; color: "red"}
+                           PropertyChanges { target: dotsGridCol; clickedFlagCol: false}
+                       }
+                   ]
+
+                   GCText {
+                       text: index
+                       anchors.centerIn: parent
+                       fontSize: (wrap.height*6)/384  //768*6/384 = 12
+                   }
+
+                   MouseArea {
+                       anchors.fill: parent
+                       onClicked: {
+                           if (dotsGridCol.state == "default") {
+                               dotsGridCol.state = "active"
+                               items.colSelected = index
+                           }
+                           else {
+                               dotsGridCol.state = "default"
+                               items.colSelected = 0
+                           }
+                           Activity.makeOtherRowInColWhite()
+                           Activity.changesInMainBoard()
+                           Activity.checkit()
+                       }
+                   }
+                }
+            }
+
+        }
+
+        Grid {
+            id: grid
+            anchors {
+                top: gridRow.bottom
+                topMargin: items.spacing
+                left: gridCol.right
+                leftMargin: items.spacing
+            }
+
+            spacing: items.spacing
+            columns: 10
+            rows: 10
+
+            Repeater {
+                id: repeater
+                model: 100
+                Rectangle {
+                    id: dots
+                    width: items.rectWidth
+                    height: items.rectHeight
+                    color: "green"
+
+                    GCText {
+                        text: (Math.floor(index/10) + 1) * (index%10 + 1)
+                        anchors.centerIn: parent
+                        fontSize: (wrap.height*6)/384  //768*6/384 = 12
+                    }
+
+                }
+            }
+        }
+
+        GCText{
+            id: instruction
+            anchors {
+                top: parent.top
+                topMargin: (wrap.height/4 - instruction.height/3)
+                left: grid.right
+                leftMargin: (wrap.width*17)/683 // 1366*17/683 = 34
+            }
+            fontSize: (wrap.height*7)/384  //768*7/384 = 14
+            text: qsTr("Instruction:\n")+
+                    qsTr("   Multiplicand x Multiplier\n")+
```

Comment:
This is not a nice way to align text. It may be aligned or not depending on the font;